assemblies/features/framework/src/main/feature/feature.xml

```diff
@@ -27,7 +27,7 @@
         <!-- mvn: and wrap: url handlers -->
         <bundle start-level="5">mvn:org.ops4j.pax.url/pax-url-aether/${pax.url.version}</bundle>
         <!-- logging -->
-        <bundle start-level="8">mvn:org.ops4j.pax.logging/pax-logging-api/${pax.logging.version}</bundle>
+        <bundle start-level="8">mvn:org.ops4j.pax.logging/pax-logging-api/${pax-logging.tesb.version}</bundle>
         <bundle start-level="8">mvn:org.ops4j.pax.logging/pax-logging-log4j2/${pax-logging-log4j2.tesb.version}</bundle>
```

Comment:
Why using two properties?

Reply:
In all previous versions, we had to patch pax-logging-log4j2 sooner or later. So I left the extra property.